src/App.tsx

```diff
@@ -1,28 +1,61 @@
-import React, { useState } from 'react';
+import React, { useCallback, useEffect, useState } from 'react';
 import './App.scss';
 import './styles/general.scss';
 import { TodoList } from './components/TodoList';
 import { CurrentUser } from './components/CurrentUser';
+import { API_URL, request } from './components/API/api';
 
 const App: React.FC = () => {
-  const [
-    selectedUserId,
-    // setSelectedUserId,
-  ] = useState(0);
+  const [selectedUserId, setSelectedUserId] = useState(0);
+  const [todos, setTodos] = useState([]);
+  const [downloadError, setDownloadError] = useState('');
+
+  useEffect(() => {
+    const getDataFromServer = async () => {
+      try {
+        const dataFromServer = await request(API_URL, '');
```

Comment:
create all the API calls (load user and todos) on api file (don't need export API_URL from api file)